### ISSUE TITLE
Ipv6Address.add does not support adding long values

### ISSUE THREAD

**Issue by maschind4** [janvanbesien/java-ipv6#27]
Hi, 

the function add(int value) support only an int value to add to a given ip address. However a long would also be feasible to add to an IpV6. 

m.

**Comment by janvanbesien** [comment]
Agreed. Are you interested in providing a patch? It would be good if both add and subtract support long values.

**Comment by maschind4** [comment]
Not nice, but a quick workaround for now

`String longNumber = value.toString();

        List<Integer> intList = new ArrayList<>();
        for (char ch : longNumber.toCharArray()) {
            intList.add(Integer.valueOf(String.valueOf(ch)));
        }

        IPv6Address retval = null;
        for (Integer intValue : intList) {
            retval = ip.add(intValue);
        }`

**Comment by sunny-shu** [comment]
@janvanbesien 
hi,janvanbesien. 
I've noticed that your project is not very well maintained. i used it in our system,do you need a maintaner to help you handle the issue and review the PR?

**Comment by janvanbesien** [comment]
All contributions (preferably via a PR) are welcome. Do you have a PR for this issue?
